packages/components/Atoms/Global/scss/settings/_color.scss

```diff
@@ -4,6 +4,7 @@ $rpl-colors: (
   'secondary': #0095ec,
   'extra_dark_neutral': #011a3c,
   'dark_neutral': #465870,
+  'dark_neutral_1': #546a7c,
```

Comment:
Hey @alan-cole as our `$rpl-colors` list is not merging with custom theme var `$rpl-colors`, so this will be overridden. It will add a breaking change to all customer sites. They will got an error because couldn't find color `dark_neutral_1`, unless they add this into their new item into their `$rpl-colors`. You can test it in https://github.com/dpc-sdp/ripple/tree/master/examples/basic-examples.

I think the merge feature shouldn't be hard. Not sure if it's ok for you to try it in [`rpl-color`](https://github.com/dpc-sdp/ripple/blob/626ecab28eaf5fd17abbb3b80df97b101874d3b6/packages/components/Atoms/Global/scss/settings/_functions.scss#L54) function. https://sass-lang.com/documentation/values/maps#add-to-a-map in this task. We should merge custom theme `$rpl-colors` into core one(need rename it to `$rpl-colors-core`?) and without delete your new color items.

All the place use `$rpl-colors` then need a review after that changes. https://github.com/dpc-sdp/ripple/search?q=%24rpl-colors&unscoped_q=%24rpl-colors

Reply:
Ah yes! Good pickup - I remember this happened with the black. I'll look at the merge technique.